storm/src/main/storm/mesos/schedulers/SchedulerUtils.java

```diff
@@ -87,7 +88,7 @@ public static boolean supervisorExists(String offerHost, Collection<SupervisorDe
         offerResourcesListPerNode.put(hostName, new ArrayList<OfferResources>());
       }
       offerResourcesListPerNode.get(hostName).add(offerResources);
-      log.info("Available resources at " + hostName + ": " + offerResources.toString());
+      log.info("Available resources at {} : {}", hostName, offerResources.toString());
```

Comment:
nit: this changed the format from:  `host.com: My Resources` to `host.com : My Resources`.  Not sure it matters.